netbout-client/src/main/java/com/netbout/client/retry/ReAlias.java

```diff
@@ -70,7 +70,7 @@ public ReAlias(final Alias orgn) {
     @RetryOnFailure(
         verbose = false, attempts = Tv.TWENTY,
         delay = Tv.FIVE, unit = TimeUnit.SECONDS
-    )
+        )
```

Comment:
@danff please see https://github.com/teamed/qulice/issues/411 - I think it was fixed